### PR TITLE
[Fluid] Nightly Build failing in Darcy Test

### LINKED ISSUE #8452: Nightly Build failing in FluidDynamics
**Description**
From last night, first time this happened
~~~
======================================================================
ERROR: testDarcyNonLinear (darcy_channel_test.DarcyChannelTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/__w/Kratos/Kratos/bin/Release/applications/FluidDynamicsApplication/tests/darcy_channel_test.py", line 86, in testDarcyNonLinear
    self.runDarcyChannelTest()
  File "/__w/Kratos/Kratos/bin/Release/applications/FluidDynamicsApplication/tests/darcy_channel_test.py", line 68, in runDarcyChannelTest
    self.runTest()
  File "/__w/Kratos/Kratos/bin/Release/applications/FluidDynamicsApplication/tests/darcy_channel_test.py", line 265, in runTest
    self.fluid_solver.FinalizeSolutionStep()
  File "/__w/Kratos/Kratos/bin/Release/KratosMultiphysics/FluidDynamicsApplication/navier_stokes_two_fluids_solver.py", line 265, in FinalizeSolutionStep
    self._GetDistanceReinitializationProcess().Execute()
RuntimeError: Error: Zero sum in skyline_lu factorization


in kratos/solving_strategies/builder_and_solvers/residualbased_block_builder_and_solver.h:495:void ResidualBasedBlockBuilderAndSolver<UblasSpace<double, boost::numeric::ublas::compressed_matrix<...>, boost::numeric::Vector >,...>::InternalSystemSolveWithPhysics(ResidualBasedBlockBuilderAndSolver::TSystemMatrixType &,...]
   kratos/solving_strategies/builder_and_solvers/residualbased_block_builder_and_solver.h:544:virtual void ResidualBasedBlockBuilderAndSolver<UblasSpace<double, boost::numeric::ublas::compressed_matrix<...>, boost::numeric::Vector >,...>::BuildAndSolve(typename TSchemeType::Pointer, ModelPart &, ResidualBasedBlockBuilderAndSolver::TSystemMatrixType &,...]
   kratos/processes/variational_distance_calculation_process.h:353:virtual void VariationalDistanceCalculationProcess<3, UblasSpace<double, boost::numeric::ublas::compressed_matrix<...>, boost::numeric::Vector >, UblasSpace<double, boost::numeric::ublas::matrix<double,...>, boost::numeric::Vector >, LinearSolver<UblasSpace<double, boost::numeric::ublas::compressed_matrix<...>, boost::numeric::Vector >, UblasSpace<double, boost::numeric::ublas::matrix<double,...>, boost::numeric::Vector >, Reorderer<UblasSpace<double, boost::numeric::ublas::compressed_matrix<...>, boost::numeric::Vector >, UblasSpace<double, boost::numeric::ublas::matrix<double,...>, boost::numeric::Vector > > > >::Execute() [TDim = 3, TSparseSpace = UblasSpace<double, boost::numeric::ublas::compressed_matrix<...>, boost::numeric::Vector >, TDenseSpace = UblasSpace<double, boost::numeric::ublas::matrix<double,...>, boost::numeric::Vector >, TLinearSolver = LinearSolver<UblasSpace<double, boost::numeric::ublas::compressed_matrix<...>, boost::numeric::Vector >, UblasSpace<double, boost::numeric::ublas::matrix<double,...>, boost::numeric::Vector >, Reorderer<UblasSpace<double, boost::numeric::ublas::compressed_matrix<...>, boost::numeric::Vector >, UblasSpace<double, boost::numeric::ublas::matrix<double,...>, boost::numeric::Vector > > >]
~~~

Happens only with CLANG for some reason

### PR DESCRIPTION
**Pull request by mrhashemi**
**Description**
[Addressing Nightly Build failing in FluidDynamics #8452](https://github.com/KratosMultiphysics/Kratos/issues/8452)

**Changelog**
darcy_channel_test.py

closes #8452

### CODE REVIEW

File: applications/FluidDynamicsApplication/tests/darcy_channel_test.py [hunk 1]
```diff
@@ -13,6 +13,7 @@ class TwoFluidNoRedistanceSolver(two_fluids_solver.NavierStokesTwoFluidsSolver):
     """
     def __init__(self, model, settings):
         super(TwoFluidNoRedistanceSolver,self).__init__(model,settings)
+        self._reinitialization_type = "none"
```
Comment (rubenzorrilla) [inline applications/FluidDynamicsApplication/tests/darcy_channel_test.py]: Couldn't we add this to the .json settings file in the TwoFluidDarcyTest folder?

Comment (mrhashemi) [reply applications/FluidDynamicsApplication/tests/darcy_channel_test.py]: > Couldn't we add this to the .json settings file in the TwoFluidDarcyTest folder?

Done.

**Review by philbucher — APPROVED** [review]
thx!

**Comment by mrhashemi** [comment]
@philbucher Windows compilation has a strange error: "Boost_INCLUDE_DIR-NOTFOUND".
Anyway, I merge this since it has nothing to do with the error.

**Comment by philbucher** [comment]
> @philbucher Windows compilation has a strange error: "Boost_INCLUDE_DIR-NOTFOUND".
> Anyway, this PR has nothing to do with the error.

#8472

**Pull request merged**